tracer/src/Datadog.Trace.Tools.dd_dotnet/Checks/Resources.cs

```diff
@@ -46,9 +46,11 @@ internal static class Resources
         public const string DdAgentChecks = "---- DATADOG AGENT CHECKS -----";
 
         public const string ContinuousProfilerEnabled = "DD_PROFILING_ENABLED is set.";
-        public const string ContinuousProfilerSsiDeployed = "DD_INJECTION_ENABLED is set. Profiler is deployed through SSI.";
+        public const string ContinuousProfilerEnabledWithHeuristics = "DD_PROFILING_ENABLED is set to 'auto'. The continuous profiler is enabled and may begin profiling based on heuristics.";
```

Comment:
Correct, it might _not_ start profiling, depending on the heuristics. e.g. if it's a short-lived process, or a process where there's no tracing happening